content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -173,6 +210,44 @@ and installing the sidecar injector webhook configuration on the remote cluster
     EOF
     {{< /text >}}
 
+    When use self signed certificates:
+
+    {{< text syntax=bash snip_id=none >}}
+    $ cat <<EOF > remote-config-cluster.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      namespace: external-istiod
+    spec:
+      profile: external
+      values:
+        global:
+          istioNamespace: external-istiod
+          configCluster: true
+        pilot:
+          configMap: true
+        istiodRemote:
+          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/:ENV:cluster=${REMOTE_CLUSTER_NAME}:ENV:net=network1
+        base:
+          validationURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/validate
+      components:
+        istiodRemote:
+          k8s:
+            overlays:
+            - kind: MutatingWebhookConfiguration
+              name: istio-sidecar-injector-external-istiod
+              patches:
+              - path: webhooks[0].clientConfig.caBundle
+                value: "${CACERT}"
+              - path: webhooks[1].clientConfig.caBundle
+                value: "${CACERT}"
+              - path: webhooks[2].clientConfig.caBundle
+                value: "${CACERT}"
+              - path: webhooks[3].clientConfig.caBundle
+                value: "${CACERT}"
```

Comment:
Instead of showing a copy of the whole config here, add a tip that just shows just the overlay part and says add this to the config.

Reply:
I was thinking about that but sometimes I found it is a bit difficult to describe it instead of a good example. will try to describe it though.